pkg/gui/controllers/helpers/refresh_helper.go

```diff
@@ -267,9 +267,11 @@ func (self *RefreshHelper) refreshReflogCommitsConsideringStartup() {
 }
 
 func (self *RefreshHelper) refreshReflogAndBranches(refreshWorktrees bool, keepBranchSelectionIndex bool) {
+	loadBehindCounts := self.c.State().GetRepoState().GetStartupStage() == types.COMPLETE
```

Comment:
@jesseduffield It's a bit unfortunate that the StartupState stuff is no longer handled only inside of `refreshReflogCommitsConsideringStartup`. I'm not sure if this bothers you; if it does, I suppose we could return a bool from the function. Not sure that makes the logic clearer, but let me know what you prefer.

Reply:
I'm not bothered, your approach is fine